integration_tests/test_page_loads.py

```diff
@@ -32,6 +33,21 @@ def auto_populate_index(populated_index):
     return populated_index
 
 
+@pytest.fixture()
+def sentry_client(client: FlaskClient) -> FlaskClient:
+    cubedash.app.config["SENTRY_CONFIG"] = {
+        "dsn": "___DSN___",
+        "include_paths": ["cubedash"],
+    }
+    return client
+
+
+def test_sentry(sentry_client: FlaskClient):
+    rv: Response = sentry_client.get("/", follow_redirects=False)
+    # Redirect to a default.
+    assert rv.location.endswith("/ls7_nbar_scene")
```

Comment:
(Already discussed on slack, but to record here)

Ideally put a comment to say what we're testing: it currently feels a little strange to have a test body that looks totally unrelated to Sentry, and which could break for unrelated reasons. 

Or even better, if possible, could we ask the sentry lib if it's initialised?